### PR TITLE
Responsiveness settings

### PR DESCRIPTION
**Pull request by MitchellQ**
This PR is working #53. This PR focuses on making the settings views responsive across all devices and in landscape mode on tablets.

There have also been a few code improvements and tidying up including
- Changed color to use HexColor
- Wrapped overlay in scaffold so that removes the error caused by no text theme
- resolve issue #54
- removed redundant centering

### CODE REVIEW

**Comment by MitchellQ** [comment]
This can't be merged until #52 is merged

**Comment by MitchellQ** [comment]
minor change to logic as it was inefficient and not producing desired output

**Comment by MitchellQ** [comment]
tested on nexus 10

File: lib/views/quiz/quiz_result.dart [hunk 1]
```diff
@@ -27,16 +27,16 @@ class QuizResult extends StatelessWidget {
                     const EdgeInsets.symmetric(vertical: 20, horizontal: 50),
                 child: Text(
                   name,
-                  style: Theme.of(context)
-                      .textTheme
-                      .headline
-                      .copyWith(fontSize: 40),
+                  style: TextStyle(
+                    fontSize: Screen.width(context) <= 350 ? 40 : 60,
```
Comment (jacksos101) [inline lib/views/quiz/quiz_result.dart]: Maybe use one of our text widgets for this

**Pull request merged**